src/osd/OSD.h

```diff
@@ -1137,11 +1137,12 @@ class OSDService {
   // -- OSD Full Status --
 private:
   Mutex full_status_lock;
-  enum s_names { NONE, NEARFULL, FULL, FAILSAFE } cur_state;  // ascending
+  enum s_names { NONE, NEARFULL, BACKFILL, FULL, FAILSAFE } cur_state;  // ascending
   const char *get_full_state_name(s_names s) {
     switch (s) {
     case NONE: return "none";
     case NEARFULL: return "nearfull";
+    case BACKFILL: return "backfill_full";
```

Comment:
I'll change all variables, config values to use "backfillfull" in all cases, for example: new_backfillfull_ratio, backfillfull_ratio, CEPH_OSD_BACKFILLFULL, mon_osd_backfillfull_ratio...